evalml/pipelines/components/transformers/samplers/base_sampler.py

```diff
@@ -59,6 +59,49 @@ def transform(self, X, y=None):
             y = infer_feature_types(y)
         return X, y
 
+    def _convert_dictionary(self, sampling_dict, y):
+        """Expects the dictionary keys to be the target values y.
```

Comment:
What do you mean by this?  Function description is a little unclear.

Reply:
Updating the docstring! 

`Converts the provided sampling dictionary from a dictionary of ratios to a dictionary of number of samples.
        Expects the provided dictionary keys to be the target values y, and the associated values to be the min:max ratios.
        Converts and returns a dictionary with the same keys, but changes the values to be the number of samples rather than ratio.`